README.md

```diff
@@ -51,7 +51,8 @@ To add new tests, edit the `*_spec.js` files in `/spec/javascripts/`.
 
 | Methods | Use | Usage (Example)|
 |---------|-----|----------------|
-|getLat   |Used to get the current latitude of the center of the map truncated to the precision the map has at current zoom.|  blurredLocation.getLat() //This would return the value in numerics|
+|getLat   |Used to get the current latitude of the center of the map truncated to the [precision]() the map has at current zoom.|  blurredLocation.getLat() //This would return the value in numerics|
 |getLon|Used to get the current latitude of the center of the map truncated to the precision the map has at current zoom|blurredLocation.getLon() //This would return the value in numerics|
 |goTo   |Takes in three parameters, namely latitude, longitude and zoom. Will set the center of map to co-ordinates input.|blurredLocation.goTo(44.51, -89.99, 13) //Will set center of map to (44.51,-89.99) with zoom set as 13|
 |addGrid  |Used to add a virtual grid to the map| blurredLocation.addGrid() //This would add a virtual grid to the map|
+|getPrecision |Used to get the precision of degrees currently occupied by one cell of the grid. This would return an integer which represents the number of decimal places occupied by a cell. For instance, 1 will mean 0.1 degrees per cell,2 will mean 0.01 degrees, and so on| blurredLocation.getPrecision() //This would return the precision of the map at the current zoom level|
```

Comment:
great, but can you show each method with parentheses `getPrecision()` to indicate they're methods and not properties? Just a small stylistic thing.